Apps/WebClient/src/ClientApp/src/views/publicCovidTest.vue

```diff
@@ -515,40 +515,40 @@ export default class PublicCovidTestView extends Vue {
                     <b-row>
                         <b-col>
                             <b-form-group
-                                label="Date of Vaccine (Any Dose)"
-                                label-for="dateOfVaccine"
-                                :state="isValid($v.dateOfVaccine)"
+                                label="Date You Were Tested for COVID-19"
+                                label-for="dateOfCollection"
+                                :state="isValid($v.dateOfCollection)"
                             >
                                 <hg-date-dropdown
-                                    id="dateOfVaccine"
-                                    v-model="dateOfVaccine"
-                                    :state="isValid($v.dateOfVaccine)"
+                                    id="dateOfCollection"
+                                    v-model="dateOfCollection"
+                                    :state="isValid($v.dateOfCollection)"
                                     :allow-future="false"
                                     :min-year="2020"
-                                    data-testid="dateOfVaccineInput"
-                                    aria-label="Date of Vaccine (Any Dose)"
+                                    data-testid="dateOfCollectionInput"
+                                    aria-label="Date You Were Tested for COVID-19"
                                     @blur="$v.dateOfBirth.$touch()"
                                 />
                                 <b-form-invalid-feedback
                                     v-if="
-                                        $v.dateOfVaccine.$dirty &&
-                                        !$v.dateOfVaccine.required
+                                        $v.dateOfCollection.$dirty &&
+                                        !$v.dateOfCollection.required
                                     "
-                                    aria-label="Invalid Date of Vaccine"
+                                    aria-label="Invalid Collection Date"
                                     data-testid="feedbackDovIsRequired"
```

Comment:
DOV is short for date of vaccination and should be updated to refer to the collection date.